initrd/bin/tpmr

```diff
@@ -235,14 +235,22 @@ tpm2_extend() {
 	while true; do
 		case "$1" in
 		-ix)
+			# store index and shift so -ic and -if can be processed
 			index="$2"
 			shift 2
 			;;
 		-ic)
+			# this is to extend the PCR with a hash of a multiword string, use array to pass multiple words in $2
+			# use array to pass multiple words in $2
```

Comment:
The "use array..." part of this comment is here twice :wink:

Reply:
removed, was misleading anyway.